src/stores/linkStore.ts

```diff
@@ -20,6 +20,11 @@ export class LinkStore {
         return this._nodes;
     }
 
+    @action
+    public changeLinkGeometry = (latLngs: L.LatLng[]) => {
+        this._link!.geometry = latLngs;
+    }
+
```

Comment:
This will conflict with linkView changes, since we there have a more generic function for this. But, since those changes are not here yet, we can't really use it yet.